openpype/scripts/otio_burnin.py

```diff
@@ -474,7 +474,10 @@ def render(self, output, args=None, overwrite=False, **kwargs):
                                "use overwrite" % output)
 
         is_sequence = "%" in output
+        keep_frame_index = kwargs.get("keep_frame_index")
+
         duration = kwargs.get("duration", 0)
+        end_frame = kwargs.get("frame_end", 0)
```

Comment:
I would replace:
```python
keep_frame_index = kwargs.get("keep_frame_index")
duration = kwargs.get("duration", 0)
end_frame = kwargs.get("frame_end", 0)
```
by 
```python
sequence_frame_id = kwargs.get("frame_id", 0)
```
and then after use it like:
```python
if is_sequence:
    output = output % sequence_frame_id
```

Of course for you need to pass the arg duration or frame_end as `frame_id` in the kwargs of the render method

Reply:
I created a frame_id variable in extract_brunins that takes the value of duration if not keep_frame_index else burnin_frame_end and passed it to otio_burnin